master/buildslaves.py

```diff
@@ -217,9 +217,14 @@ def __init__(self, name, **kwargs):
 
 # Create an HVM EC2 large latent build slave
 class ZFSEC2BuildSlave(ZFSEC2Slave):
-    def __init__(self, name, **kwargs):
+    def __init__(self, name, arch="amd64", **kwargs):
+        instance_types = {
+            "amd64": "c5d.large",
+            "arm64": "a1.xlarge"
```

Comment:
These instances are only available in a few regions for now (us-west-2 seems the closest to everything else)
How should this be handled?

Reply:
fixed in f197c4e

Reply:
I lied, c6g is available in us-west-1 but it lacks ephemeral storage.
c6gd has nvme storage available but is [only available](https://aws.amazon.com/about-aws/whats-new/2020/07/announcing-new-amazon-ec2-instances-powered-aws-graviton2-processors/) in us-east-* and us-west-2

Reply:
The way things are now we'll need it to be available in us-west-1.  But if we're only doing compilation testing we don't need anything fancy.

Reply:
Ok, I switched to c6g which is available in us-west-1 but lacks NVMe and updated the bootstrap script to handle the case where those were missing.